dnsprovider/pkg/dnsprovider/providers/aws/route53/route53.go

```diff
@@ -32,6 +32,10 @@ const (
 	ProviderName = "aws-route53"
 )
 
+var (
+	MaxBatchSize int
```

Comment:
Ah - this value needs to be set to 900, otherwise when we use this in other places the value is 0

i.e. `var MaxBatchSize = 900`

Reply:
Of course.  Nice catch.